### PR TITLE
temperature conversions

### PR DESCRIPTION
**Pull request by MaarcooC**
Added 6 functions for temperature conversion between Celsius, Fahrenheit, and Kelvin with input validation.

### CODE REVIEW

**Comment by MaarcooC** [comment]
added a function that checks if a number is perfect.

**Review by darwinz — CHANGES_REQUESTED** [review]
Thanks for submitting a PR!  Please add proper unit tests in the appropriate `tests` directory and location.  Also, your code needs to be linted and tests passing first, so please run PHPCS (`vendor/bin/phpcs -n`)

**Comment by MaarcooC** [comment]
Sorry for that, I now added proper unit tests and ran phpcs.

File: tests/Conversions/ConversionsTest.php [hunk 1]
```diff
@@ -85,4 +86,60 @@ public function testSpeedConversion()
...
+
+    public function testFahrenheitToKelvin()
+    {
+        $this->assertEquals(273.15, FahrenheitToKelvin(32));
+        $this->assertEquals(373.15, FahrenheitToKelvin(212));
+        $this->expectException(\Exception::class);
+        $this->expectExceptionMessage('Temperature (Fahrenheit) must be a number');
+        FahrenheitToKelvin("non-numeric");
+    }
+}
```
Comment (darwinz) [inline tests/Conversions/ConversionsTest.php:145]:
```suggestion
}

```

Comment (darwinz) [reply tests/Conversions/ConversionsTest.php:145]: This is a code style issue that the linter found

**Review by MaarcooC — COMMENTED** [review]
added unit tests and phpcs

**Review by darwinz — CHANGES_REQUESTED** [review]
Can you please fix the comments to be formatted as doc comments?  See [here](https://github.com/TheAlgorithms/PHP/blob/master/Sorting/BubbleSort2.php#L3C1-L10C4) for an example.

**Review by darwinz — APPROVED** [review]
Looks great.  Thank you!

**Pull request merged**